### DISCUSSION TITLE
REST API: Expose events of a customer

### DISCUSSION THREAD

**Discussion by MacLake** [pretix/pretix#4698] [Ideas]
In `GET /api/v1/organizers/(organizer)/customers/(identifier)/` we can only get some basic information about a customer, like name, e-mail, phone (see https://docs.pretix.eu/en/latest/api/resources/customers.html).

 In the web interface there is a list of events the customer participated in: `/control/organizer/xxx/customer/XXXXXXX/` We would like to access these data through the REST API.

**Comment by raphaelm** [comment]
The equivalent to the view in the backend would be to get the list of all orders:
https://docs.pretix.eu/en/latest/api/resources/orders.html#get--api-v1-organizers-(organizer)-orders-

And filter with the `customer` parameter
